src/components/button/Button.component.scss

```diff
@@ -0,0 +1,7 @@
+.button {
```

Comment:
Will `.button` only apply in this specific component? How do we scoping classes?